### PR TITLE
Add an example of a random calling pipeline

### PR DESCRIPTION
**Pull request by reedacartwright**
Random call is a useful classroom technique and can be implemented using command line tools.  I have added a challenge to the pipefilter lesson that contains an example.

### CODE REVIEW

File: 03-pipefilter.md [hunk 1]
```diff
@@ -516,3 +516,29 @@ so this matches all the valid data files she has.
...
+> 10,Douglas,black
+> ~~~
+>
+> The following command line will randomly choose one student from the roster.
+>
+> ~~~ {.bash}
+> $ cat roster.txt | tail -n +2 | cut -d, -f2 | shuf -n 1
+> ~~~
+>
+> Can you explain how this pipe works?
```
Comment (gvwilson) [inline 03-pipefilter.md]: ...to which the student answers "yes", and is done :-)

**Comment by reedacartwright** [comment]
If shuf is not installed, there are alternative solutions:

http://stackoverflow.com/questions/9245638/select-random-lines-from-a-file-in-bash

http://stackoverflow.com/questions/448005/whats-an-easy-way-to-read-random-line-from-a-file-in-unix-command-line

**Comment by gdevenyi** [comment]
Shuf will definitely not be installed for most learners, can you implement an alternate version please.

**Comment by gdevenyi** [comment]
No response from original author, closing.

If you have time, please re-submit using "sort -R"

**Pull request closed without merging**